andes/routines/tds.py

```diff
@@ -1108,3 +1090,22 @@ def check_criteria(self):
         res = deltadelta(self.system.dae.x[self.system.SynGen.delta_addr],
                          self.config.ddelta_limit)
         return res
+
+    def _csv_output(self):
```

Comment:
Consider renaming this for clarity. `_csv_output` reads like outputting to a CSV file.

Consider `_csv_data_to_dae` or similar.

Reply:
Do you want to add a test case under `tests` by the way?

Reply:
The changed lines are covered by following tests. Do you mean a test on the helper function itself? https://github.com/CURENT/andes/blob/7f56cc9ea4c16d9d58f5b440984f7ee2e6fd1018/tests/test_output.py#L78-L129